batch7crud-roll7/src/main/java/com/lftechnology/batch7crud/filter/UserFilter.java

```diff
@@ -0,0 +1,38 @@
+package com.lftechnology.batch7crud.filter;
+
+import com.lftechnology.batch7crud.controller.CommonConstants;
+
+import javax.servlet.*;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpSession;
+import java.io.IOException;
+
+/**
+ * Created by leapfrog on 1/27/16.
+ */
+public class UserFilter implements Filter {
+  private String LOGIN_ACTION_URI;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename this field "LOGIN_ACTION_URI" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS00116)